DBConnector.py

```diff
@@ -0,0 +1,57 @@
+from sqlalchemy import *
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine, update
+from sqlalchemy.orm import declarative_base, sessionmaker
+
+Base = declarative_base()
+DATABASE_URL = 'postgresql://postgres:Password123!@localhost'
+db_engine = create_engine(DATABASE_URL)
+Session = sessionmaker(bind=db_engine)
+session = Session()
+
+
+class Files(Base):
+    __tablename__ = 'Files'
+    id = Column(Integer, primary_key=True)
+    filename = Column(String(255), unique=True)
+
+    def get_record(filepath):
+        record = session.query(Files).filter(Files.filename==filepath).first()
+        return record
+    
+    def create_record(filepath):
```

Comment:
I've made DTO classes for _create_record_ methods. Seems that you changed it to passing needed parameters to _create_record_ methods. Please, make appropriate changes in Main